### ISSUE TITLE
Cannot import SelectItem in TypeScript

### ISSUE THREAD

**Issue by bear-bibeault** [primefaces/primereact#2983]
### Describe the bug

Using 8.1.1

The Get Started states that you can import `SelecItem` via:
```
import { SelectItem } from 'primereact/api';
```
But TS complains: `Module '"primereact/api"' has no exported member 'SelectItem'.`, and indeed, `api.d.ts` contains no such type.

Attempting to import directly via `import { SelectItem } from "primereact/selectitem";` also fails with TS complaining:
```
File name '/Users/xyz/projects/aw-next/aw-ui-portal/node_modules/primereact/selectitem/selectitem.d.ts' differs from already included file name '/Users/xyz/projects/aw-next/aw-ui-portal/node_modules/primereact/selectitem/SelectItem.d.ts' only in casing.
  The file is in the program because:
    Imported via '../selectitem/SelectItem' from file '/Users/xyz/projects/aw-next/aw-ui-portal/node_modules/primereact/dropdown/dropdown.d.ts' with packageId 'primereact/selectitem/SelectItem.d.ts@8.1.1'
    Imported via "primereact/selectitem" from file '/Users/xyz/projects/aw-next/aw-ui-portal/src/features/istrike/IStrikeServiceSelector.tsx'ts(1149)
dropdown.d.ts(5, 39): File is included via import here. 
```

However, `SelectItem` is not exported by `dropdown.d.ts` so it cannot be imported from there.

How does one go about importing `SelectItem`? I'd rather not resort to using `any`.

### Reproducer

https://codesandbox.io/s/primereact-test-forked-vtvy4x?file=/src/index.tsx

### PrimeReact version

8.1.1

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

Attempting to import `SelectItem` in various ways, including the manner shown in the Getting Started document, fails in various ways as noted in the report.

### Expected behavior

Ability to import `SelectItem` as described in the Getting Started document.

**Comment by melloware** [comment]
Fixed already in master for 8.2.0 see this ticket: https://github.com/primefaces/primereact/pull/2945

**Comment by melloware** [comment]
Actually let me look at this

**Comment by melloware** [comment]
What OS are you running on as well?

**Comment by bear-bibeault** [comment]
macOS Mojave

**Comment by melloware** [comment]
OK good I am pretty sure my fix for https://github.com/primefaces/primereact/pull/2945 will fix your issue in 8.2.0

**Comment by bear-bibeault** [comment]
Thank you!
